### PR TITLE
OSDOCS#10325: Added optional tags parameter in install config

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-10325

Link to docs preview: https://75757--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-restricted-networks-installer-provisioned-vsphere.html#installation-installer-provisioned-vsphere-config-yaml_installing-restricted-networks-installer-provisioned-vsphere

Review:
- [x] QE has approved this change. @WenXinWei 
- [x] SME has approved this change. @rvanderp3

### CODE REVIEW

File: modules/installation-installer-provisioned-vsphere-config-yaml.adoc [hunk 1]
```diff
@@ -114,18 +116,20 @@ You can specify the path of any datastore that exists in a datastore cluster. By
 If you must specify VMs across multiple datastores, use a `datastore` object to specify a failure domain in your cluster's `install-config.yaml` configuration file. For more information, see "VMware vSphere region and zone enablement".
 ====
 <7> Optional: Provides an existing resource pool for machine creation. If you do not specify a value, the installation program uses the root resource pool of the vSphere cluster.
-<8> The vSphere disk provisioning method.
+<8> Optional: Each VM created by {product-title} is assigned a unique tag that is specific to the cluster. The assigned tag enables the installation program to identify and remove the associated VMs when a cluster is decommissioned. You can list of up to ten tag IDs.
```
Comment (rvanderp3) [inline modules/installation-installer-provisioned-vsphere-config-yaml.adoc:119]:
```suggestion
<8> Optional: Each VM created by {product-title} is assigned a unique tag that is specific to the cluster. The assigned tag enables the installation program to identify and remove the associated VMs when a cluster is decommissioned. You can list of up to ten additional tag IDs to be attached to the VMs provisioned by the installer.
```

Comment (snarayan-redhat) [reply modules/installation-installer-provisioned-vsphere-config-yaml.adoc]: @rvanderp3 can you approve if the changes look fine now?

**Comment by rvanderp3** [comment]
LGTM once https://github.com/openshift/openshift-docs/pull/75757#discussion_r1620769891is addressed

**Comment by jhou1** [comment]
@sgaoshang @WenXinWei please review, thanks

File: modules/installation-installer-provisioned-vsphere-config-yaml.adoc [hunk 2]
```diff
@@ -42,7 +42,7 @@ networking:
     hostPrefix: 23
   machineNetwork:
   - cidr: 10.0.0.0/16
-  networkType: OVNKubernetes <9>
+  networkType: OVNKubernetes <11>
```
Comment (jeana-redhat) [inline modules/installation-installer-provisioned-vsphere-config-yaml.adoc:45]: In case a merge reviewer notices that this is out of order: this is out of scope and should be addressed as a separate installer docs cleanup effort and Shubha doesn't need to worry about it :slightly_smiling_face:

**Review by jeana-redhat — COMMENTED** [review]
Looks familiar :laughing: Nice work, just one nit from me.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/installation-installer-provisioned-vsphere-config-yaml.adoc (hunk 2 above)
Comment (snarayan-redhat) [reply modules/installation-installer-provisioned-vsphere-config-yaml.adoc:45]: Thank you for the thoughtful note @jeana-redhat . Since I have merge rights, I will take a note :smile:
